src/lib/memory/numa_memory_resource.cpp

```diff
@@ -3,7 +3,7 @@
 #include <string>
 
 #if HYRISE_NUMA_SUPPORT
-#define NUMA_MEMORY_RESOURCE_ARENA_SIZE 1llu << 30u
+constexpr auto NUMA_MEMORY_RESOURCE_ARENA_SIZE = 1llu << 30u;
```

Comment:
Big fan of the various removals of `#define`.

Reply:
But same as above: I really don't like those short cut definition of numbers. But probably to large here anyways.

Reply:
Here, we can express it as `1024 * ...`